src/applications/personalization/dashboard/actions/claims.jsx

```diff
@@ -205,3 +219,64 @@ export function getClaimsV2(options = {}) {
     });
   };
 }
+
+export function getLighthouseClaims(options = {}) {
+  const recognizedOptions = ['poll', 'pollingExpiration'];
```

Comment:
I wouldn't suggest using polling with the new API. The old API required you to use polling because it was not synchronous but the new one is.